### ISSUE TITLE
New install: transmission failing due to trouble with libcurl

### ISSUE THREAD

**Issue by DiagonalArg** [Linuxbrew/homebrew-core#4539]
New install, CentOS 7. After the install, I tried installing some other things, including w3m and socat (reported in another issue). Also, transmission, which failed on libcurl. Error is:

brew install transmission
==> Downloading https://github.com/transmission/transmission-releases/raw/094777d/transmission-2.92.tar.xz                                                                                       \

Already downloaded: /home/user/.cache/Homebrew/transmission-2.92.tar.xz
==> ./configure --prefix=/home/user/.linuxbrew/Cellar/transmission/2.92_1 --disable-mac --without-gtk --disable-nls                                                                           \

Last 15 lines from /home/user/.cache/Homebrew/Logs/transmission/01.configure:
checking for library containing gethostbyname... none required
checking for library containing quotacursor_skipidtype... no
checking for pkg-config... /home/user/.linuxbrew/opt/pkg-config/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for LIBCURL... configure: error: Package requirements (libcurl >= 7.15.4) were not met:

No package 'libcurl' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables LIBCURL_CFLAGS
and LIBCURL_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

READ THIS: https://github.com/Linuxbrew/brew/blob/master/docs/Troubleshooting.md#troubleshooting
Please do not report this issue to Homebrew/brew or Homebrew/core, which support macOS only.

**Comment by DiagonalArg** [comment]
`brew config` is below.  No problems show up in `brew doctor`

HOMEBREW_VERSION: 1.3.1
ORIGIN: https://github.com/Linuxbrew/brew
HEAD: 09208fb2af972b19a46f89ec8321232943e4c8d4
Last commit: 3 weeks ago
Core tap ORIGIN: https://github.com/Linuxbrew/homebrew-core
Core tap HEAD: be0a4fdd01822dcf1b5b352ca96262c8ae014403
Core tap last commit: 2 hours ago
HOMEBREW_PREFIX: /home/user/.linuxbrew
HOMEBREW_REPOSITORY: /home/user/.linuxbrew/Homebrew
HOMEBREW_CELLAR: /home/user/.linuxbrew/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://linuxbrew.bintray.com
CPU: 40-core 64-bit broadwell
Homebrew Ruby: 2.0.0-p643 => /usr/local/bin/ruby2.0
Clang: N/A
Git: 2.7.4 => /usr/bin/git
Perl: /usr/bin/perl
Python: /home/user/.linuxbrew/opt/python/libexec/bin/python => /home/user/.linuxbrew/Cellar/python/2.7.14/bin/python2.7
Ruby: /usr/bin/ruby
Java: openjdk version "1.8.0_144"
Kernel: Linux 3.10.0-514.16.1.el7.x86_64 x86_64 GNU/Linux
OS: CentOS Linux release 7.4.1708 (Core) 
Host glibc: 2.17
/usr/bin/gcc: 4.8.5
glibc: 2.23
gcc: 5.4.0
xorg: N/A

**Comment by iMichka** [comment]
Hi. Thanks for reporting. This should be as simple as adding:

> depends_on "curl" => :build unless OS.mac?

to the list of the decencies for that formula. On OS X curl is present on the system. On Linux, it may be there, but we need to explicitly tell brew to look for it. Would you like to open a pull request for that?

**Comment by DiagonalArg** [comment]
Ok, that was simple enough and worked.  Hm, I guess it's time for me to learn some git, as I'm not sure how do a pull request... Give me a little time...

**Comment by iMichka** [comment]
`brew bump-formula-pr` is used when there is a new version of a formula. This is not what you are looking for.

You could follow the instructions here: https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request.html
You will need to adapt them a little bit for Linuxbrew, as the repository here is a different one, but in principle that should be fine.

Or you can use the github  GUI and edit the file directly.

**Comment by DiagonalArg** [comment]
Done.  Hopefully, it's right!

**Comment by DiagonalArg** [comment]
Hm.  While the following tests are fine on my machine, CircleCI says:

```
linkage --test transmission - brew-test-bot.x86_64_linux
failed: brew linkage --test transmission

test transmission - brew-test-bot.x86_64_linux
failed: brew test transmission --verbose
```

**Comment by iMichka** [comment]
Fixed by #4553. Thanks!
